### PR TITLE
Allow get Algolia index name from an entity class

### PR DESCRIPTION
**Pull request by qdequippe**
Expose `getAlgoliaIndexName` method and rename old method `getInternalAlgoliaIndexName`.

Please check issue #127

### CODE REVIEW

**Comment by julienbourdeau** [comment]
This change definitely makes sense.

I have been working on a new version fo this bundle, it's tagged in beta. I have taken this into accounted and added a simple way to get the full index name (https://github.com/algolia/AlgoliaSearchBundle/commit/c0bd165750df569b068213a654e36aff9987ecec)
If you find time to test it, I'd love your feedback.

I should release a new version of the 2.0 branch very soon.

**Comment by alcaeus** [comment]
@julienbourdeau I saw you changed the base to 2.x - beware that the signature change of `getAlgoliaIndexName` is a BC break.

File: Indexer/Indexer.php [hunk 1]
```diff
@@ -466,6 +466,26 @@ public function getAlgoliaIndexName($entity_or_class)
...
     }
 
+    /**
+     * @param ObjectManager $em
+     * @param               $entityName
+     *
+     * @return string
+     * @throws NotAnAlgoliaEntity
+     */
+    public function getAlgoliaIndexName(ObjectManager $em, $entityName)
```
Comment (OskarStark) [inline Indexer/Indexer.php]: as @alcaeus said, this is a BC break

**Comment by qdequippe** [comment]
Hi, since this feature is available in 3.x version of this bundle, is this PR is still needed? (I can close it if it's the case)

**Comment by julienbourdeau** [comment]
I'd like to release a last version vo the 2.x branch with the pending PRs since there are still people using Sf 2.8 or other version 3.4.

Although, after one year of this PR being opened I can really ask you to fix the branch 😅 (I think you updated your master branch and that was the branch used for this PR). Feel free to either update or close.

Just to know, did you migrate to v3?

**Comment by qdequippe** [comment]
@julienbourdeau yes I am migrating to v3, I will close this PR, it is no longer necessary.

**Pull request closed without merging**